kubernetes-model-generator/kubernetes-model-core/src/main/java/io/fabric8/kubernetes/api/model/GenericKubernetesResource.java

```diff
@@ -56,7 +57,8 @@ public class GenericKubernetesResource implements HasMetadata {
   @JsonProperty("kind")
   private String kind;
   @JsonProperty("metadata")
-  private ObjectMeta metadata;
+  @JsonInclude(value = Include.CUSTOM, valueFilter = ObjectMeta.class)
```

Comment:
Is this supposed to ignore this field when the ObjectMeta is "blank"-> `metadata.equals(new ObjectMeta())` ?

Reply:
Yes